src/material/datepicker/calendar-body.ts

```diff
@@ -31,6 +31,8 @@ export type MatCalendarCellClassFunction<D> = (
   view: 'month' | 'year' | 'multi-year',
 ) => MatCalendarCellCssClasses;
 
+export const FOCUS_ACTIVE_CELL_DELAY = 20;
```

Comment:
Add a comment that explains what this constant is and where the value came from (as detailed as possible)